doc/help/CP.md

```diff
@@ -0,0 +1,7 @@
+kdb-cp(1) -- Copy keys within the key database
+================================
+
+## USAGE
+`kdb cp <source> <dest>`  
```

Comment:
You are right, fixed in 2bb39a21db44ef57f585d029d24773b85813036a

Also interesting is copying keys into themselves (recursively). Just play around, I am interested in what you find!